pkg/apis/core/validation/validation.go

```diff
@@ -1433,6 +1435,14 @@ func validateCSIPersistentVolumeSource(csi *core.CSIPersistentVolumeSource, fldP
 		allErrs = append(allErrs, field.Required(fldPath.Child("driver"), ""))
 	}
 
+	if len(csi.Driver) > 63 {
+		allErrs = append(allErrs, field.TooLong(fldPath.Child("driver"), csi.Driver, 63))
+	}
+
+	if !driverNameRexp.MatchString(csi.Driver) {
+		allErrs = append(allErrs, field.Invalid(fldPath.Child("driver"), csi.Driver, fmt.Sprintf("Driver name must match : %s", driverNameRexp)))
```

Comment:
Don't other regex based validation errors include example valid names? Not everyone speaks regex fluently

Reply:
@liggitt fixed, PTAL